caffe2/python/tutorials/Loading_Pretrained_Models.ipynb

```diff
@@ -63,14 +63,20 @@
    ],
    "source": [
     "%matplotlib inline\n",
+    "from __future__ import unicode_literals\n",
+    "from __future__ import absolute_import\n",
+    "from __future__ import print_function\n",
+    "from __future__ import division\n",
     "from caffe2.proto import caffe2_pb2\n",
     "import numpy as np\n",
     "import skimage.io\n",
     "import skimage.transform\n",
     "from matplotlib import pyplot\n",
     "import os\n",
     "from caffe2.python import core, workspace, models\n",
-    "import urllib2\n",
+    "from future.standard_library import install_aliases\n",
```

Comment:
`future.standard_library` is non-standard

Reply:
fixed it! thank you!